evap/evaluation/models.py

```diff
@@ -559,6 +559,13 @@ def general_contribution_has_questionnaires(self):
 
     @property
     def all_contributions_have_questionnaires(self):
+        if is_prefetched(self, "contributions"):
+            if not self.contributions:
+                return False
+
+            if is_prefetched(self.contributions[0], "questionnaires"):
+                return all(len(contribution.questionnaires) > 0 for contribution in self.contributions)
```

Comment:
Again, maybe we should go for `not any` to make it easier to verify that the two approaches do the same?